app/controllers/comments_controller.rb

```diff
@@ -13,9 +13,10 @@ def create
     @new_comment.post_id = @post.id
     @new_comment.update_comments_counter
     if @new_comment.save
-      redirect_to "/users/#{@post.user_id}/posts/#{@post.id}", notice: 'Success!'
+      redirect_to "/users/#{@post.user_id}/posts/#{@post.id}", flash[:notice] = 'Your comment is saved!'
```

Comment:
- [ ] Same here. While creating a comment, I'm getting the error of `no implicit conversion of symbol into string`. Please fix this.

![Screenshot from 2022-02-25 22-59-07](https://user-images.githubusercontent.com/22434660/155761068-dbb5ca1d-13b2-4d2f-9093-667ca7df9ead.png)

Reply:
Thanks for your time and feedback. I'll fix it ASAP.